### PR TITLE
Dataset statistics pedigree update

### PR DESCRIPTION
**Pull request by joankosev**
Update dataset statistics "families by pedigree" table
- make pedigrees clickboxes bigger
- add the family count to the modal as well
- change mouseover highlight from tr to td

### CODE REVIEW

File: src/app/variant-reports/common-reports-pedigree-cell/common-reports-pedigree-cell.component.ts [hunk 1]
```diff
@@ -8,5 +9,10 @@ import { PedigreeCounter } from '../variant-reports';
 })
 export class CommonReportsPedigreeCellComponent {
   @Input() public pedigree: PedigreeCounter;
+  @ViewChild(PedigreeComponent) public pedigreeComponent: PedigreeComponent;
   public pedigreeMaxWidth = 300;
+
```
Comment (IvoStefanov) [inline src/app/variant-reports/common-reports-pedigree-cell/common-reports-pedigree-cell.component.ts:14]: Write test for openModal

File: src/app/pedigree/pedigree.component.ts [hunk 2]
```diff
@@ -43,10 +44,14 @@ export class PedigreeComponent {
     });
   }
```
Comment (IvoStefanov) [inline src/app/pedigree/pedigree.component.ts:46]: Write test for openModal

**Review by IvoStefanov — CHANGES_REQUESTED** [review]
Write unit tests

File: src/app/variant-reports/common-reports-pedigree-cell/common-reports-pedigree-cell.component.ts [hunk 3]
```diff
@@ -8,5 +9,10 @@ import { PedigreeCounter } from '../variant-reports';
...
+  public openModal(): void {
```
Comment (IvoStefanov) [inline src/app/variant-reports/common-reports-pedigree-cell/common-reports-pedigree-cell.component.ts:15]: Write test for openModal

File: src/app/pedigree/pedigree.component.ts [hunk 4]
```diff
@@ -43,10 +44,14 @@ export class PedigreeComponent {
...
 
-  public openModal(pedigreeModal): void {
+  public openModal(): void {
```
Comment (IvoStefanov) [inline src/app/pedigree/pedigree.component.ts:47]: Write test for openModal

**Review by IvoStefanov — CHANGES_REQUESTED** [review]
Write unit tests

**Review by livovachkov — APPROVED** [review]

**Pull request merged**